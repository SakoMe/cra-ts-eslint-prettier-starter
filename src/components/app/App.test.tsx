import { render } from '@testing-library/react';

import { App } from './';

describe('<App /> spec', () => {
  it('renders App', () => {
    const { asFragment } = render(<App />);

    const app = asFragment();
    expect(app).toMatchSnapshot();
  });

  it('renders starter text', () => {
    const { getByText } = render(<App />);

    const title = getByText(/Start here/i);
    expect(title).toBeInTheDocument();
    expect(title.tagName.toLowerCase()).toEqual('h1');
  });
});
