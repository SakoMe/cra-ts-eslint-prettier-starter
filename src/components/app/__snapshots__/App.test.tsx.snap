// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> spec renders App 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      Start here
    </h1>
  </div>
</DocumentFragment>
`;
